clash_lib/src/lib.rs

```diff
@@ -286,8 +286,8 @@ async fn start_async(opts: Options) -> Result<(), Error> {
     }
 
     runners.push(Box::pin(async move {
-        info!("receiving shutdown signal");
         shutdown_rx.recv().await;
+        info!("receiving shutdown signal");
```

Comment:
why changing this

Reply:
> why changing this

I thought this log should after received shutdown signal